src/main/java/com/performance/demo/performance/AdbPerformanceCollector.java

```diff
@@ -36,10 +29,17 @@ public class AdbPerformanceCollector extends PerformanceCollector implements IDr
 
     private final String bundleId = getAppPackage();
     private final GeneralParser generalParser;
-    private final String cpuCommand = generateCpuCommand();
-    private final String errorOutput = String.format("No process found for: %s%n", bundleId);
     private final String pidCommand = String.format(PerformanceTypes.PID.cmdArgs, bundleId);
 
+    private String pid;
+    private String cpuCommand;
+    private String memCommand;
+    private String memCommand2;
+    private String nifCommand;
```

Comment:
this field can be converted to a local variable in generateCommands() method as it's used only here